Main.sublime-menu

```diff
@@ -4,11 +4,6 @@
         "caption": "Tools",
         "children":
         [
-            {
```

Comment:
Why are you putting this directly under Tools?  I see your commit message as "Fix the error menu items"...what are you fixing?